level2/instance.tf

```diff
@@ -23,14 +23,17 @@ resource "aws_security_group" "private" {
   description = "Allow SSH from VPC"
   vpc_id      = data.terraform_remote_state.level1.outputs.vpc_id
 
+  # (No longer needed since we are implementing SSM) 
+  # ingress {
+  #   description = "SSH from VPC"
+  #   from_port   = 22
+  #   to_port     = 22
+  #   protocol    = "tcp"
+  #   cidr_blocks = [data.terraform_remote_state.level1.outputs.vpc_cidr] #Using datasource "Pulling from level1" 
+  # }
```

Comment:
Commenting-out code instead of removing is a bad style. I suggest that you remove this commented-out code block completely, as well as other code blocks as well.